pyproject.toml

```
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "tflite-support"
-version = "0.0.0.dev0"
+version = "0.1.0"
```

@DuvanMarrugo14 considering latest version is 0.4.4 (https://github.com/tensorflow/tflite-support) I think we should name ours 1.0.0 And rename it: `xc-tflite-support`
```suggestion
version = "1.0.0"
```